assembly/src/release/docs/user-guide.html

```diff
@@ -98,7 +98,7 @@ <h2>Access the web demos</h2>
 </p>
 <pre>
     cd bin
-    activemq.bat console xbean:examples/conf/activemq-demo.xml
+    activemq.bat start xbean:../examples/conf/activemq-demo.xml
```

Comment:
NB: `activemq console` is actually valid (at least for Unix shell script). `activemq.bat` doesn't really take any action arg. Maybe we should clean the arg.

Reply:
Thank you for the review!
You are correct that the activemq.bat file does not take any arguments directly, but it does pass the provided arguments to the jar using %*. For reference, see this line: 
https://github.com/apache/activemq/blob/6b6f5fbe24550452cd09aab9c4d7094cf782a04f/assembly/src/release/bin/activemq.bat#L99 
Running ``activemq.bat`` without ``start`` argument will not start the broker.